readme.md

```diff
@@ -41,7 +41,7 @@ var theBody = document.querySelectorAll("body");
 theBody.style.backgroundColor = "red";
 ```
 
-> Your answer...
+document.body should be used instead of querySelectorAll.
```

Comment:
:+1: Yes!

`querySelectorAll` always returns an array of elements, even when that array includes just one element. This can be fixed with `document.body`, `document.querySelectorAll("body")[0]`, or `document.querySelector("body")`.